cdap-distributions/src/packer/cdap-sdk-ubuntu16-with-uri.json

```diff
@@ -172,11 +172,44 @@
     {
       "type": "chef-solo",
       "remote_cookbook_paths": "/var/chef/cookbooks",
-      "run_list": "recipe[openssh],recipe[cdap::sdk]",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "permit_root_login": "without-password",
+            "password_authentication": "no",
+            "use_dns": "no"
+          }
+        }
+      },
+      "except": ["cdap-cloud-sandbox-azure"]
+    },
+    {
+      "type": "chef-solo",
+      "remote_cookbook_paths": "/var/chef/cookbooks",
+      "run_list": "recipe[openssh]",
+      "skip_install": true,
+      "json": {
+        "openssh": {
+          "server": {
+            "client_alive_interval": "120",
```

Comment:
I was only concerned about the Ami and its requirements, so didn't want to change it. If you don't think it's an issue I'll make a common change. I need to change the value to 180 also, to avoid a warning in the certification